debugging/book-library/script.js

```diff
@@ -90,14 +99,26 @@ function render() {
 
     //add delete button to every row and render again
     let delButton = document.createElement("button");
-    delBut.id = i + 5;
-    cell5.appendChild(delBut);
-    delBut.className = "btn btn-warning";
-    delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
+    delButton.id = `del-${i}`;
+    delButton.className = "btn btn-warning";
+    delButton.innerHTML = "Delete";
+    cell5.appendChild(delButton);
+
+    delButton.addEventListener("click", function () {
+      console.log('ll')
```

Comment:
I wonder why do you need this console.log()?

Reply:
I was checking to see if the event was working. (Removed now)